src/actions/instances.ts

```diff
@@ -266,6 +266,29 @@ export const setInstanceDataRefValueOnRemote = throttle((instance: InstanceState
 		oscQueryBridge.sendPacket(writePacket(message));
 	}, 100);
 
+export const setInstanceParameterMetaOnRemote = (instance: InstanceStateRecord, param: ParameterRecord, value: string): AppThunk =>
+	() => {
+		const message = {
+			address: `${param.path}/meta`,
+			args: [
+				{ type: "s", value }
+			]
+		};
+
+		oscQueryBridge.sendPacket(writePacket(message));
+	};
+
+export const restoreDefaultParameterMetaOnRemote = (instance: InstanceStateRecord, param: ParameterRecord): AppThunk =>
+	() => {
+		const message = {
+			address: `${param.path}/meta`,
+			args: [
+				{ type: "s", value: "" }
```

Comment:
here is the "restore" value which is currently `""`. Wonder if we want this to be something else as really one might want to just "clear" the metadata but it would always be interpreted as "restore" I guess?

Reply:
yeah.. clear is currently `{}"`or `null`.. some valid json that doesn't actually do anything... so `""` in a way makes sense because it isn't valid json?? maybe?